packages/core/src/rum/instrumentation/DdRumResourceTracking.tsx

```diff
@@ -255,8 +274,12 @@ export class DdRumResourceTracking {
                     this._datadog_xhr.spanId
                 );
                 this.setRequestHeader(ORIGIN_HEADER_KEY, ORIGIN_RUM);
-                this.setRequestHeader(SAMPLING_PRIORITY_HEADER_KEY, '1');
-                this.setRequestHeader(SAMPLED_HEADER_KEY, '1');
+                this.setRequestHeader(
```

Comment:
I am not sure we have to change this value. @xgouchet ?

Reply:
That is what is stated in the RFC but actually keeping the header value to `'1'` does not have any impact once the `trace_id` and `span_id` are removed, so I'll wait for Xavier's answer.

Reply:
Now is clear, after I saw your Slack discussion. I guess we need to change this header also in case the resource trace is sampled out.